### DISCUSSION TITLE
密码忘记了怎么找回

### DISCUSSION THREAD

**Discussion by 361639072** [qd-today/qd#367] [Q&A] [answered]
密码忘记了，没有设置邮箱，请大佬解答

**Comment by a76yyyy** [comment, marked as answer]
自建框架请通过注册一个新用户, 通过后台容器内执行 `python ./chrole.py your@email.address admin` 方式将新用户设置为管理员, 退出重新登录新账号即可修改原管理员账号密码

**Reply by 1779750448** [reply]
很好，谢谢

**Reply by kfyu26** [reply]
忘记是不是关闭了注册，现在没有看到注册链接了
